### PR TITLE
User can view items on Bespoken Words.

### PR DESCRIPTION
**Pull request by Stevenlee731**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    name: 'Calhoun',
+    description: 'Russel & Hazel, updates the classic coptic journal with a contemporary cover.',
+    button: 'Add to Cart',
+    learnMore: 'Learn More',
+    link: 'https://google.com',
+    image: 'images/10067179gMetallicNotebook_600.jpg'
+  }
+]
+
+items.forEach(function(item) {
```
Comment (thebearingedge) [inline main.js:70]: Hi Steve, you should create a separate function to render the `$item` and call that function in your `forEach`.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: main.js (hunk 1 above)
Comment (thebearingedge) [reply main.js:70]:
```js
var $items = document.querySelector('#items')

items.forEach(function (item) {
  var $item = renderItem(item)
  $items.appendChild($item)
})
```

Comment (Stevenlee731) [inline main.js]: added the changes!

Comment (Stevenlee731) [reply main.js:70]: Made the changes!

**Review by Stevenlee731 — COMMENTED** [review]
Made the changes you requested

**Pull request merged**